src/borg/archiver.py

```diff
@@ -3582,8 +3582,8 @@ def define_borg_mount(parser):
                                help='interpret PATH as command and store its stdout. See also section Reading from'
                                     ' stdin below.')
         subparser.add_argument('--paths-from-stdin', action='store_true',
-                               help='read DELIM-separated list of paths to backup from stdin. Will not '
-                                    'recurse into directories.')
+                               help='read DELIM-separated list of paths to backup from stdin. Exclusive option - no recursion'
+                                    ', no excludes, ``-x`` does not apply.')
```

Comment:
Shall we just say "It will back up all files given, no more, no less."?

Reply:
Sounds good to me. Is the v1 still relevant for more than a few weeks? No point bothering you with 1.2 PRs, if v2 is imminent.

Reply:
1.2.x will be maintained for a longer time, most users have that in production, it is in stable dists and borg2 will still take a while (and even after it is released, people will need some time to adopt/migrate).

Reply:
#5248 (and there was no pressure).